index.js

```diff
@@ -157,12 +157,17 @@ function extractCssResults(dest, stylesheetsModule, stylesheetsPort) {
   return function () {
     return new Promise(function (resolve, reject) {
       var Elm = require(dest);
+      var parts = stylesheetsModule.split('.');
+      var worker = Elm[parts.shift()];
 
-      if (!(stylesheetsModule in Elm)){
-        return reject(missingEntryModuleMessage(stylesheetsModule, Elm));
-      }
+      parts.forEach(function (part) {
+        if (!worker || !worker[part]){
+          return reject(missingEntryModuleMessage(stylesheetsModule, Elm));
+        }
+        worker = worker[part];
+      })
 
-      var worker = Elm[stylesheetsModule].worker();
+      worker = worker.worker();
```

Comment:
This code is pretty confusing. Can I ask that you add comments, or at least simplify it?